README.md

```diff
@@ -81,7 +81,7 @@ $ git clone git@github.com:redox-os/redox.git --origin upstream --recursive
 $ cd redox/
 
 # Install/update dependencies
-$ sudo <your package manager> install llvm make nasm virtualbox virtualbox-dkms qemu qemu-kvm
+$ sudo <your package manager> install make nasm virtualbox qemu
```

Comment:
I was removing the two extra things that are not included by default in every package manager's archive, and should be installed automatically with the main package anyway.